src/index.html

```diff
@@ -9,6 +9,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo__link" href="#home">
```

Comment:
Where is block `logo`? Fix BEM classes

Reply:
is it really necessary to add div.logo? Should i just rename like "logo-link" and "logo-pic" according to BEM?